### DISCUSSION TITLE
有群吗

### DISCUSSION THREAD

**Discussion by Hanchers** [WeNeedHome/SummaryOfLoanSuspension#124] [Q&A]
大佬们整个qq群或微信群呗

**Comment by KisaraKira** [comment]
整qq wx群是生怕晶哥逮不住你们吗

**Reply by pengMaster** [reply]
qq wx 群太容易进去了

**Comment by caifugao110** [comment]
tg群组有了通知下

**Reply by WeihanLi** [reply]
楼下看到一个 https://t.me/xujiayinhuanqian

**Reply by zouya1990** [reply]
优秀

**Reply by CPAJim** [reply]
前段时间有个马某某用电报群，发表中国人的独立宣言，要建立中国大陆临时国会，还不是被抓了，估计是无期。
我建了一个只需要邮箱就可以加入的群，不需要手机验证或实名认证，所有群聊是公开透明的，不能在群里发隐私信息，可以考虑一下。群组链接：
https://groups.google.com/u/0/g/chineseindependence?pli=1

**Comment by xameless** [comment]
整wx，qq是准备一网打尽是吧

**Comment by CPAJim** [comment]
就是电报群都不安全，别说QQ群了。
前段时间有个马某某用电报群，发表中国人的独立宣言，要建立中国大陆临时国会，还不是被抓了，估计是无期。
我建了一个只需要邮箱就可以加入的群，不需要手机验证或实名认证，所有群聊是公开透明的，不能在群里发隐私信息，可以考虑一下。群组链接：
https://groups.google.com/u/0/g/chineseindependence?pli=1

**Reply by HsinWang94** [reply]
Google groups可以的

**Reply by diaoyudong** [reply]
这个里面的内容很怪

**Reply by pengMaster** [reply]
你这独立宣言看着挺吓人啊 兄得

**Reply by qingyun95** [reply]
这是什么乱七八糟的，湾湾搞的东西吧

**Reply by DSbajRv5Ol4X** [reply]
别用自己的86手机号注册任何敏感网站，别用老王vpn，别用国内的输入法。电报不隐藏手机号，用国内的网名，这不就是等着被抓吗？

**Comment by sdtime-io** [comment]
国内整个群一网打尽
